packages/admin-next/dashboard/src/components/common/list-summary/list-summary.tsx

```diff
@@ -61,6 +61,7 @@ export const ListSummary = ({
             className="cursor-default whitespace-nowrap"
             title={title}
```

Comment:
nit: I would remove this - it's a bit annoying since we already have a tooltip on hover